doc/sphinxman/source/build_planning.rst

```diff
@@ -1447,3 +1447,53 @@ Ways to refer to |PSIfour| in text, in order of decreasing goodness:
 
   * **NOT** ``PSI4`` or ``PSI``
 
+
+.. _`faq:localaddon`:
+
+How to use a local Add-On repository in the Psi4 build
+------------------------------------------------------
+
+For each Add-On, |PSIfour| pulls source from a specific online Git
+repository and a specific tag/branch/commit in it. This ensures success
+of the |PSIfour| build, reproducibility of the runtime results, and
+freedom for continued upstream development. Sometimes, you're the one
+doing that development, and you need the CMake superbuild to pull source
+from a local path rather than the approved codeset.
+
+Find the ``CMakeLists.txt`` governing the target Add-On in
+:source:`external` and make changes analogous to the below::
+
+    #GIT_REPOSITORY https://github.com/jturney/ambit
+    #GIT_TAG 1.0
+    DOWNLOAD_COMMAND ""
+    SOURCE_DIR "/path/to/ambit-directclone"
+
+If you're changing the |PSIfour| repo codebase between compiles, there's
+nothing more to do as CMake will handle the code rebuild deps for you.
+
+If you're changing the local Add-On repo codebase between compiles,
+CMake *does not* know when ``libaddon.a/so/dylib`` needs rebuilding. It
+is recommended that the |PSIfour| build be initially configured with
+``-DBUILD_SHARED_LIBS=ON`` (easier to notice changes). And to trigger
+Add-On library rebuild, ``rm -rf {objdir}/external/upstream/addon/``
+and ``rm -rf {objdir}/stage/{prefix}/share/cmake/AddOn``. This should
```

Comment:
Would it be possible to add this to our CMake/make `make clean add-on`?

Reply:
Do we have _any_ `make clean` capabilities? I've never trusted them enough to experiment.

Those would be nice, but, in any case, these particular commands are hitting the install folders or the written-upon-cmake-configure folders, so they're not a usual use of `make clean`. Even devs should be hitting this FAQ rarely.